.NET/Microsoft.Recognizers.Definitions.Common/Japanese/NumbersDefinitions.cs

```diff
@@ -113,6 +113,7 @@ public static class NumbersDefinitions
         {
             '十'
         };
+      public const string UnitsRegex = @"(k|mil|t|g|b|km|ml|m|ミリリットル(入れら)?|キロメートル|メートル|ミリメート)";

```

Comment:
What do these represent? All multipliers? If so, the regex should be named NumberMultiplierRegex as in other languages and reference the one in BaseUnit if re-using all values.

Reply:
Agreed. We'll do that.